ArchiSteamFarm/IPC/Controllers/Api/BotController.cs

```diff
@@ -283,7 +283,7 @@ public async Task<ActionResult<GenericResponse>> InputPost(string botNames, [Fro
 	[HttpGet("{botNames:required}/Inventory/{appID}/{contextID}")]
 	[ProducesResponseType<GenericResponse<IReadOnlyDictionary<string, BotInventoryResponse>>>((int) HttpStatusCode.OK)]
 	[ProducesResponseType<GenericResponse>((int) HttpStatusCode.BadRequest)]
-	public async Task<ActionResult<GenericResponse>> InventoryGet(string botNames, uint appID, ulong contextID) {
+	public async Task<ActionResult<GenericResponse>> InventoryGet(string botNames, uint appID, ulong contextID, [FromQuery(Name = "l")] string language = "english") {
```

Comment:
All other query parameters are written out (see `RedeemPointsPost`, `GitHubWikiPageGet` and `PluginsGet`). I'd recommend removal of the `(Name = "l")`-part.